Sources/SmileID/Classes/Metadata/MetadataManager.swift

```diff
@@ -4,14 +4,18 @@ import Network
 import UIKit
 
 protocol MetadataProvider {
-    func collectMetadata() -> [MetadataKey: CodableValue]
+    func collectMetadata() -> [Metadatum]
 }
 
 public class MetadataManager {
     public static let shared = MetadataManager()
 
+    private struct StaticEntry {
+        var value: CodableValue
+        var date: Date
```

Comment:
We can do the same here
```suggestion
        var date: Date = Date().toISO8601WithMilliseconds()
```